.github/workflows/tests.yml

```
@@ -58,7 +60,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        devices: [BackblazeTest, DOSpacesTest, LinodeTest, LocalTest, S3Test, WasabiTest]
+        devices: [BackblazeTest, DOSpacesTest, ExoscaleTest,LinodeTest, LocalTest, S3Test, WasabiTest]
```

Can you add a space here
```suggestion
        devices: [BackblazeTest, DOSpacesTest, ExoscaleTest, LinodeTest, LocalTest, S3Test, WasabiTest]
```